libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -670,7 +670,7 @@ - (void)logoutUser:(SFUserAccount *)user reason:(SFLogoutReason)reason {
     
     // Before starting actual logout (which will tear down SFRestAPI), first unregister from push notifications if needed
     __weak typeof(self) weakSelf = self;
-    [[SFPushNotificationManager sharedInstance] unregisterSalesforceNotificationsWithCompletionBlock:user completionBlock:^void() {
```

Comment:
Objc and Swift overlook here.  Looks like the original method was this as well.

Reply:
Can this go back to passing the user parameter?

Reply:
yes